CHANGELOG.md

```diff
@@ -1,11 +1,16 @@
-# [1.0.0] 3 Aug 2022 - Unreleased
+# [1.0.0 - 16 Aug 2022](https://github.com/SimformSolutionsPvtLtd/flutter_calendar_view/tree/1.0.0)
```

Comment:
Please change the date of release to today. If we are going to release it today.